sim_pipeline/RSP.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import mpl_toolkits.axisartist.floating_axes as floating_axes
+from mpl_toolkits.axisartist.grid_finder import (FixedLocator, MaxNLocator,
+                                                 DictFormatter)
+from matplotlib.transforms import Affine2D
+import pickle
+import lsst.daf.butler as dafButler
+import lsst.geom as geom
+# Source injection
+from lsst.pipe.tasks.insertFakes import _add_fake_sources
+from sim_pipeline.image_simulation import gsobj_true_flux
+import galsim
```

Comment:
is galsim a dependence of the sim-pipeline package? If not, please add it, as well as all the other packages that are imported here